CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ This project includes all static files of `swagger-ui` under `sanic_openapi/ui/`
 
 ## Code Style
 
-In `dev` requirements, it already including some modules related to code style:
+In `dev` requirements, it already includes some modules related to code style:
```

Comment:
suggested:
In `dev` requirements, some modules related to code style are already included: